.teamcity/Gradle_Check/configurations/SmokeTests.kt

```diff
@@ -14,6 +14,7 @@ class SmokeTests(model: CIBuildModel, stage: Stage, testJava: JvmCategory, task:
     params {
         param("env.ANDROID_HOME", "/opt/android/sdk")
         param("env.JAVA_HOME", buildJavaHome())
+        param("env.GRADLE_BUILD_JAVA_HOME", buildJavaHome())
```

Comment:
There's no need to inject this from the environment,  via the build logic. The test can just use `AvailableJavaHomes` to find an appropriate JVM to run the `gradle/gradle` build, like we do for other tests that need a JVM that is a specific version.